src/core_plugins/metrics/public/visualizations/components/gauge_vis.js

```diff
@@ -0,0 +1,193 @@
+import React, { Component, PropTypes } from 'react';
```

Comment:
Gauge is responsible for handling just the metrics (text portion) of the visualization where the GaugeVis is responsible for handling the SVG portion.